app/root.tsx

```diff
@@ -13,6 +13,18 @@ export const meta: MetaFunction = () => {
     charSet: "utf-8",
     httpEquiv: "X-UA-Compatible",
     content: "IE=edge",
+    description: "Uniting the best analytical minds in the space to build the future of crypto analytics.",
+    "og:image": "https://bounty.metricsdao.xyz/social/twitter/metricsdao_banner.png",
+    "og:description": "Uniting the best analytical minds in the space to build the future of crypto analytics.",
+    "og:url": "https://bounty.metricsdao.xyz",
+    "og:title": "Metrics DAO",
+    "og:type": "website",
+    "twitter:card": "summary_large_image",
+    "twitter:domain": "metricsdao.xyz",
+    "twitter:url": "https://metricsdao.xyz",
+    "twitter:title": "Metrics DAO",
+    "twitter:description": "Uniting the best analytical minds in the space to build the future of crypto analytics.",
+    "twitter:image": "https://bounty.metricsdao.xyz/social/twitter/metricsdao_banner.png",
```

Comment:
I don't like relying on this "not in our future plans" app. We can upload it in this app

Reply:
haha good call, I am actually about to switch the images anyway